tests/php/i18n/i18nTextCollectorTest.php

```diff
@@ -709,6 +712,43 @@ public function testCollectFromEntityProvidersInWebRoot()
         );
     }
 
+    public function testCollectFromORM()
+    {
+        // note: Disable _fakewebroot manifest for this test
+        $this->popManifests();
+
+        // update config because test can't load config from actual class
+        Config::inst()->update(TestDataObject::class, 'db', [
+            'MyProperty' => 'Varchar',
+            'MyUntranslatedProperty' => 'Text'
+        ]);
+        Config::inst()->update(TestDataObject::class, 'has_one', [
+            'HasOneRelation' => Member::class
+        ]);
+        Config::inst()->update(TestDataObject::class, 'has_many', [
+            'HasManyRelation' => Member::class
+        ]);
+        Config::inst()->update(TestDataObject::class, 'many_many', [
+            'ManyManyRelation' => Member::class
+        ]);
```

Comment:
@GuySartorelli I have added a test for this, but as you can see I had to update the config for the class manually. For whatever reason the config was not loaded from the class statics. Any idea why? Are the class names/namespaces in the tests handled differently?

Reply:
bump